finder-app/writer.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+writefile=$1
+writestr=$2
+
+if [ $# -ne 2 ]
+then
+	echo "Error"
```

Comment:
@Ruthvik-Chanda think about how you could make this error message more useful to the caller